pkg/core/resources/model/resource.go

```diff
@@ -54,35 +54,24 @@ type KDSFlagType uint32
 
 const (
 	// KDSDisabledFlag is a flag that indicates that this resource type is not sent using KDS.
-	KDSDisabledFlag = KDSFlagType(0)
-
-	// ZoneToGlobalFlag is a flag that indicates that this resource type is sent from Zone CP to Global CP.
-	ZoneToGlobalFlag = KDSFlagType(1)
-
-	// GlobalToAllZonesFlag is a flag that indicates that this resource type is sent from Global CP to all zones.
-	GlobalToAllZonesFlag = KDSFlagType(1 << 2)
-
-	// GlobalToAllButOriginalZoneFlag is a flag that indicates that this resource type is sent from Global CP to
-	// all zones except the zone where the resource was originally created. Today the only resource that has this
-	// flag is ZoneIngress.
-	GlobalToAllButOriginalZoneFlag = KDSFlagType(1 << 3)
-)
-
-const (
-	// GlobalToZoneSelector is selector for all flags that indicate resource sync from Global to Zone.
-	// Can't be used as KDS flag for resource type.
-	GlobalToZoneSelector = GlobalToAllZonesFlag | GlobalToAllButOriginalZoneFlag
-
-	// AllowedOnGlobalSelector is selector for all flags that indicate resource can be created on Global.
-	AllowedOnGlobalSelector = GlobalToAllZonesFlag
-
-	// AllowedOnZoneSelector is selector for all flags that indicate resource can be created on Zone.
-	AllowedOnZoneSelector = ZoneToGlobalFlag | GlobalToAllButOriginalZoneFlag
+	KDSDisabledFlag      = KDSFlagType(0)
+	ConsumedByZoneFlag   = KDSFlagType(1)
```

Comment:
What are the cases, when
* the resource type has `ConsumedByZoneFlag` but **not** `ProvidedByGlobalFlag`
* the resource type has `ConsumedByGlobalFlag` but **not** `ProvidedByZoneFlag`

Reply:
Ah, I see, so basically `ConsumedByX` doesn't necessarily mean "consumed through KDS", it means "resource type on X can be created through KDS, API, Kube store". I think it would be nice to add docs for these flags

Reply:
Sounds good!

Reply:
done